### DISCUSSION TITLE
[Idea] allow # in product/category name

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#36442] [Ideas]
### The problem

product/category name don't allow #

### The solution

allow #

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
I think it should be okay to consider this character as the correct one in the product and category name. I'm ok with this change 👍🏻 What do you think @Hlavtox ?

**Comment by jf-viguier** [comment]
product/category names do not allow the characters ^<>;=#{}
I don't know why, maybe we should consider to allow some others characters.

**Comment by Hlavtox** [comment]
I definitely agree, I think that there is no reason to disallow any character, no? :-)

Urls need to be built clean, query strings also, database queries are escaped, XML data also needs to by escaped. Why block ^<>;=#{} anywhere?
